### DISCUSSION TITLE
Podman Autostart cointener

### DISCUSSION THREAD

**Discussion by jarx88** [louislam/dockge#71] [Ask for Help] [answered]
Podman containers do not automatically start after a system restart. Is there a possibility to, after a few seconds from the system start, send a start command to all managed containers?

Currently, I bypass this by creating a service in `/etc/systemd/system/` like
```
[Unit]
Description=dockge homepage container
Wants=syslog.service
[Service]
Restart=always
ExecStartPre=/bin/sleep 30
ExecStart=/usr/bin/podman start -a homepage
ExecStop=/usr/bin/podman stop -t 10 homepage
[Install]
WantedBy=multi-user.target
```
and `systemctl enable --now dockge-homepage.service`

**Comment by grencez** [comment, marked as answer]
Doesn't `podman-restart.service` do this for containers that are configured to always restart?

Or `rc-update add podman boot` if using OpenRC.

**Reply by jarx88** [reply]
Thanks a lot! I've created the service `/etc/systemd/system/podman-restart.service`

```
[Unit]
Description=Podman Start All Containers With Restart Policy Set To Always
Documentation=man:podman-start(1)
StartLimitIntervalSec=0
Wants=network-online.target
After=network-online.target

[Service]
Type=oneshot
RemainAfterExit=true
Environment=LOGGING="--log-level=info"
ExecStart=podman $LOGGING start --all --filter restart-policy=always
ExecStop=/bin/sh -c 'podman $LOGGING stop $(podman container ls --filter restart-policy=always -q)'

[Install]
WantedBy=default.target
```

and work perfect
